src/components/BookCard.js

```diff
@@ -16,7 +16,7 @@ function BookCard({
         <h4>{author}</h4>
         <div className="edit">
           <p>Comments</p>
-          <button type="submit" onClick={() => dispatch(removeBook(id))}>Remove</button>
+          <button id="removeBtn" type="submit" onClick={() => dispatch(removeBook(id))}>Remove</button>
           <p>Edit</p>
         </div>
```

Comment:
- [ ] For all text in the Book card, kindly use the font family, and size provided in the template.